### ISSUE TITLE
Translations are not loaded from composer repository

### ISSUE THREAD

**Issue by pkerling** [codeigniter4/CodeIgniter4#2120]
**Describe the bug**
The documentation says that when using composer and requiring `codeigniter4/translations`, the translations should be automatically picked up and that there is no need to copy them to the `app` directory.
This does not currently work unfortunately.

**CodeIgniter 4 version**
4.0.0-beta4

**Affected module(s)**
language, autoloader

**Expected behavior, and steps to reproduce if appropriate**
* Make new composer installation and require `codeigniter4/translations`
* Set default locale to anything other than English
* In the demo controller, try for example `var_dump(lang('Validation.noRuleSets'))`
* It will display the English message.

The cause is that the Language module tries to look up the file `Language/{$locale}/{$file}.php`, but the `translations` composer repository registers the `Translations` namespace as the `Language` directory, which means that codeigniter will try to look for `Translations/Language/{$locale}/{$file}.php` in the package. The correct file would be `Translations/{$locale}{$file}.php`. Adjusting the path that is looked up to `{$locale}/${file}.php` does allow it to work, but it breaks the look-up inside the application and is of course not the real solution (no idea how to solve this cleanly).

**Context**
 - OS: Linux
 - Web server lighttpd 1.4.54
 - PHP version 7.3.6

**Comment by jim-parry** [comment]
oops! I cn fix this next week, when I am vack from vacation :-/

**Comment by jim-parry** [comment]
Corrected the composer autoload config in the translations repo.
You should be able to do a composer update in your project now, and have the translations properly detected.
Can you confirm? Thanks.

**Comment by pkerling** [comment]
Unfortunately, composer version 4.0.0-beta.4 does not seem to contain the fix. I think you may have released the wrong commit?

**Comment by jim-parry** [comment]
Hmm - I see that. I re-released beta.4, but the binaries were not updated.
I have made a new beta.5 for the translations repo only, which will hopefully correct this :-/

**Comment by jim-parry** [comment]
Not hearing anything contrary in 4 days, I presume the fix works.

**Comment by pkerling** [comment]
No, it does not seem to work. As far as I can tell, the namespace you put into the composer.json is irrelevant for the lookup actually. Did it work on your end?

**Comment by jim-parry** [comment]
Gah - I thought it worked for me. WIll recheck :-/

**Comment by jim-parry** [comment]
Finally nailed this, as far as I can tell.
Nothing to change inside the CodeIgniter4 repo, only the composer.json in codeigniter4/translations.
I moved the translations to their own namespace, rather than trying to map them inside the CodeIgniter one; CI's Autoloader "conveniently" strips the CodeIgniter namespace(s) from the composer autloaded set, since it is loaded earlier. Arghhh.
I tested this on a fresh download of the appstarter, then composer required codeigniter4/translations, and joy ensued :)
